.github/workflows/deploy.yml

```diff
@@ -49,9 +32,20 @@ jobs:
           images: asmilex/iv
 
       - name: build y push
+        if: |
```

Comment:
Estás usando lo mismo que @modejota y de la misma manera. Si vais a hacer algo en colaboración, y más si es algo que no se está evaluando *avisadlo*.

Reply:
No estamos colaborando, pero supongo que habremos acabado en el mismo repositorio al poner en Google "How to check changed files Github Action" :sweat_smile:

Reply:
Pues mencionadlo también, por favor.

Reply:
Es el primer resultado que sale cuando buscas `check changed files github actions`.

Reply:
Si no tienes claro cuál es el problema, la solución que te va a dar tampoco es la mejor. Google te responde a lo que le preguntas, pero como decía @mapachana el otro día, "Basura dentro, basura fuera". En este caso no era ese el problema que queríais resolver, porque no estáis en un monorepo. Simplemente, son cosas independientes que se pueden hacer en actions independientes.